CondTools/Hcal/src/HcalPFCutsHandler.cc

```diff
@@ -0,0 +1,33 @@
+#include "CondTools/Hcal/interface/HcalPFCutsHandler.h"
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/HcalDetId/interface/HcalGenericDetId.h"
```

Comment:
```suggestion
```
These do not seem to be used
Certainly missing:
-  #include "FWCore/MessageLogger/interface/MessageLogger.h"